JavaScript/github/public/js/main.js

```diff
@@ -0,0 +1,93 @@
+var topics = [];
+
+// this API Key will only work on Algorithmia's website; get your own key at https://algorithmia.com/user#credentials
+var apiKey = 'sim+m8/xp0+/xKjzpBOd0cM/fzh1'
```

Comment:
:bulb: Perhaps just a quick comment here about where this key lives and the permissions it has.

Reply:
Updated the google sheet with the key info, would you still like me to update the comment?